http://code.google.com/p/orayta/issues/detail?id=4

--- origsrc/orayta/Orayta.pro	2010-06-06 03:05:54.000000000 -0500
+++ src/orayta/Orayta.pro	2010-06-06 03:40:54.834124100 -0500
@@ -86,8 +86,7 @@ TRANSLATIONS = Hebrew.ts \
                French.ts
 
 # Install binaries
-binary.path = /usr/local/bin
-binary.files = Orayta
+target.path = /usr/bin
 
 # Install books
 books.path = /usr/share/Orayta/
@@ -109,7 +108,7 @@ menu.files = Orayta.desktop
 trans.path = /usr/share/Orayta/
 trans.files = Hebrew.qm
 
-INSTALLS += binary
+INSTALLS += target
 INSTALLS += books
 INSTALLS += icon
 INSTALLS += trans
