--- origsrc/orayta/Orayta.pro	2012-06-14 15:09:52.000000000 -0500
+++ src/orayta/Orayta.pro	2012-06-14 15:11:24.770470800 -0500
@@ -7,12 +7,6 @@ TARGET = orayta
 TEMPLATE = app
 INCLUDEPATH += $$PWD
 
-    #for test only
-    DEFINES += MOBILE
-    DEFINES += android
-    CONFIG += MOBILE
-    CONFIG += android
-
 android {
 #    INSTALL_PATH = "/sdcard/Orayta/"
 #    INSTALL_BOOKS_PATH = "/sdcard/Orayta/Books/"
